sharktank/pyproject.toml

```diff
@@ -9,7 +9,7 @@ authors = [
 ]
 description = "SHARK layers and inference models for genai"
 readme = "README.md"
-license = {text = "Apache-2.0"}
+license = "Apache-2.0"
```

Comment:
Oh wait, this doesn't build?

```
         File "/tmp/pip-build-env-twhtcvpo/overlay/lib/python3.11/site-packages/setuptools/dist.py", line 432, in _finalize_license_expression
          raise InvalidConfigError(
      setuptools.errors.InvalidConfigError: License classifiers have been superseded by license expressions (see https://peps.python.org/pep-0639/). Please remove:
      
      License :: OSI Approved :: Apache Software License
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error
```

https://github.com/nod-ai/shark-ai/actions/runs/14837045158/job/41650593053?pr=1383#step:6:92

Reply:
Seems I forgot to push half of my changes (the ones addressing the second warning). Pushed it, hope that fixes the error.